### PR TITLE
Improve Fashion MNIST tutorial per feedback

### PR DESCRIPTION
**Pull request by alsrgv**

### CODE REVIEW

File: fashion_mnist/README.md [hunk 1]
```diff
@@ -369,21 +374,13 @@ After a few minutes, you should see training progress.  It will be faster compar
...
-```
-
-![image](https://user-images.githubusercontent.com/16640218/53537203-072b8280-3abe-11e9-8ffa-3cb1b1ed5922.png)
+Open the browser and load `http://<ip-address-of-vm>:6006/`:
 
-Now, open the browser and load `http://<ip-address-of-vm>:6006/`:
+![image](https://user-images.githubusercontent.com/16640218/54213792-3ec50200-44a2-11e9-9c7d-fdf9ab1bf94f.png)
 
-![image](https://user-images.githubusercontent.com/16640218/53537263-35a95d80-3abe-11e9-9f88-45c15a4eab11.png)
+By default, TensorBoard shows metric comparison based on the number of epochs.  This is shown on the chart above.  To compare training time it takes to achieve certain accuracy, select **RELATIVE** in the *Horizontal Axis* selector:
```
Comment (sblotner) [inline fashion_mnist/README.md]: achieve --> achieve a

File: fashion_mnist/fashion_mnist_solution.py [hunk 2]
```diff
@@ -157,8 +158,8 @@
 
 # Train the model. The training will randomly sample 1 / N batches of training data and
 # 3 / N batches of validation data on every worker, where N is the number of workers.
-# Over-sampling of validation data helps to increase probability that every validation
-# example will be evaluated.
+# Over-sampling of validation data, which helps to increase the probability that every
```
Comment (sblotner) [inline fashion_mnist/fashion_mnist_solution.py:161]: If this is an instruction or description, recommend Over-sampling --> This example configures oversampling

**Review by sblotner — COMMENTED** [review]
A few minor edits, but overall lots of good context and examples

File: fashion_mnist/fashion_mnist_solution.py (hunk 2 above)
Comment (alsrgv) [reply fashion_mnist/fashion_mnist_solution.py:161]: I'll skip this because it would require re-taking screenshots.
